fenix/app/src/main/res/values/strings.xml

```diff
@@ -2304,4 +2304,18 @@
     <string name="download_languages_item_content_description_not_downloaded_state">Download</string>
     <!-- Content description (not visible, for screen readers etc.): For a language list item that is selected. -->
     <string name="download_languages_item_content_description_selected_state">Selected</string>
+
+    <!-- Debug drawer -->
+    <!-- The title of the Tab Tools feature in the Debug Drawer. -->
+    <string name="debug_drawer_tab_tools_title" tools:ignore="UnusedResources">Tab Tools</string>
+    <!-- The title of the tab count section in Tab Tools. -->
+    <string name="debug_drawer_tab_tools_tab_count_title">Tab count</string>
+    <!-- The active tab count category in the tab count section in Tab Tools. -->
+    <string name="debug_drawer_tab_tools_tab_count_normal">Active</string>
```

Comment:
nit: perhaps in another PR could `debug_drawer_tab_tools_tab_count_normal` also be updated to `debug_drawer_tab_tools_tab_count_active`?

Reply:
dang. yeah. thanks for catching this.

https://bugzilla.mozilla.org/show_bug.cgi?id=1872881